src/Microsoft.AspNetCore.DataProtection.Extensions/DataProtectionProvider.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.IO;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
See comment here: https://github.com/aspnet/DataProtection/pull/138/files#r58582323

This might not matter